src/vaccination_sites/dto/searchVaccinationSites-dto.ts

```diff
@@ -0,0 +1,5 @@
+export class SearchDto {
```

Comment:
Class name duy nhất được export thì phải trùng file name
```suggestion
export class SearchVaccinationSitesDto.ts {
```